sdcm/keystore.py

```diff
@@ -29,9 +29,13 @@
 
 
 class KeyStore:  # pylint: disable=too-many-public-methods
-    def __init__(self):
-        self.s3: S3ServiceResource = boto3.resource("s3")
-        self.s3_client: S3Client = boto3.client("s3")
+    @property
+    def s3(self) -> S3ServiceResource:
+        return boto3.resource("s3")
+
+    @property
+    def s3_client(self) -> S3Client:
+        return boto3.client("s3")
```

Comment:
how this change helps?

Reply:
It returns new object each time the `s3_client` attribute is called.
It allows to avoid object sharing in different threads.